src/scripts/site.js

```diff
@@ -546,8 +546,8 @@ var AppInfo = {};
             }
         }
 
-        for (var index = 0, length = externalPlugins.length; index < length; index++) {
-            list.push(externalPlugins[index]);
+        if (window.NativeShell) {
+            list = list.concat(window.NativeShell.getPlugins());
```

Comment:
Don't we want to keep the web players just in case?

Reply:
And they are kept in place. I just removed an array that is never filled with plugins. The list array contains all current plugins, including the web ones.